packages/yew/Cargo.toml

```diff
@@ -135,7 +135,6 @@ web_sys = [
 doc_test = []
 wasm_test = []
 httpbin_test = []
-echo_server_test = []
```

Comment:
Please revert

Reply:
This goes in `yew-services`. In fact `httpbin_test` should be removed